LayoutTests/platform/glib/TestExpectations

```diff
@@ -1162,6 +1162,7 @@ webkit.org/b/213699 imported/w3c/web-platform-tests/mediacapture-record/MediaRec
 webkit.org/b/213699 http/wpt/mediarecorder/set-srcObject-MediaStream-Blob.html [ Pass Failure ]
 webkit.org/b/213699 http/wpt/mediarecorder/MediaRecorder-audio-bitrate.html [ Pass Failure ]
 webkit.org/b/213699 http/wpt/mediarecorder/MediaRecorder-AV-audio-video-dataavailable.html [ Timeout Failure ]
+webkit.org/b/282534 http/wpt/mediarecorder/MediaRecorder-audio-samplingrate-change.html [ Failure ]
```

Comment:
Is it really failing? Couldn't find any red WPE/GTK EWS test run in the backlog.

Reply:
Yes. They don’t show as red because I pushed a fix before the tests completed and they shows as cancelled instead. 

look at the one canceled like
https://ews-build.webkit.org/#/builders/34/builds/58736

Reply:
If they weren’t failing, the expectation would be wrong and it would now fail :)